packages/ringcentral-widgets/lib/OAuthBase/index.js

```diff
@@ -133,20 +135,20 @@ export default class OAuthBase extends RcModule {
   }
 
   @required
-  async prepareOAuth() {}
+  async prepareOAuth() { }
 
   @required
-  async destroyOAuth() {}
+  async destroyOAuth() { }
 
   @required
-  openOAuthPage() {}
+  openOAuthPage() { }
 
 
   get oAuthUri() {
     const extendedQuery = qs.stringify({
       force: true,
       localeId: this._locale.currentLocale,
-      ui_options: 'hide_remember_me hide_tos',
+      ui_options: ['hide_remember_me', 'hide_tos'].concat(this._extralUIOptions).join(' ')
```

Comment:
Can we provide default options, and allow developer to override options. For example, some developers don't want to add `hide_remember_me`